modules/uhttp/README.md

```diff
@@ -39,15 +39,15 @@ func registerHTTP(service service.Host) []uhttp.RouteHandler {
 }
 ```
 
-HTTP handlers are set up with filters that inject tracing, authentication information etc. into the
+HTTP handlers are set up with inbound middlewares that inject tracing, authentication information etc. into the
```

Comment:
I see! that is why `rpc.WithInboundMiddleware` is singular! Should I change it? it will effect `inboundMiddlewareChainBuilder.AddMiddlewares` and `uhttp.WithInboundMiddlewares`